# Project Overview & Scope

## Standards and Methods
[Click here](http://www.genomes2fields.org/docs/Genomes_to_Fields_Phenotype_Handbook.pdf) to download Genomes to Fields phenotyping handbook guidelines


### Project Summary Table

|                                   |    2014    |   2015    |   2016    |   2017    |
|:----------------------------------|:----------:|:---------:|:---------:|:---------:|
| No. of experiments             |	23       | 	28       |    40     |    43     |  
| No. of unique locations           |	19       | 	24       |    34     |    38     |  
| No. of states/provinces           |   13       |   16      |    19     |    23     |  
| No. of Principal  Investigators   |   19       |	25       |    29     |    32     |  
| No. of plots                      |	12,678   |	13,650   | 	19,360   | 	21,186   |  
| No. of unique inbreds             |	380      |	553      |	313      |	295      |
| Ex-PVP                            |	25       |	55       | 	76       |	83       |
| Biparental derived                |	302      |	301      |	44       |	42       |
| Diverse	                        |    53      |	197      | 	193      |	170      |


## 2014 Experiment
The 2014 GxE Experiment consisted of a large hybrid trialing effort and a smaller inbred trialing effort. For our first year of field experiments, 25 principal investigators in 14 states and one Canadian province tested over 900 hybrids, deployed as groups of 250 hybrids with two replications at each location. With 23 participating locations, a total of 12,678 plots were grown. As part of the 250 hybrids, a set of 10 diverse hybrids was tested across all locations; other hybrids were selected based on reasonable maturity for specific locations. Except for the set of common hybrids, testers included LH198, LH195, LH185, CG102 and PB80. In addition, 31 inbred lines were replicated 2 to 12 times over 15 locations for a total of 1,598 plots. Inbred lines tested in 2014 include recombinant inbred lines (RILs), Expired Plant Variety Protection (exPVP) inbreds and lines from collaborator breeding programs.

## 2015 Experiment
For the second year of evaluations, the GxE experiment expanded to include Ohio, Kansas, and South Dakota, for a total of 17 states and one Canadian province overseen by 28 principal investigators. Data were collected from 28 hybrid experiments and 25 inbred experiments in 27 unique locations. For the hybrid experiment, effort was made to maintain continuity with 2014 while expanding the diversity of materials included in 2015 experiments and increasing overlap among locations. Most locations grew 500 hybrid plots, with a total of 13,650 plots. The design was modified from 2014 to increase the number of common hybrids to 45 hybrids in all locations plus 15 regional hybrid checks in Northern, Central, and Southern regions, each replicated twice per experiment for 120 entries. The remaining 380 hybrids were mostly unreplicated, except for the most northern and most southern locations where additional hybrids were included with two replications. Each hybrid was included in at least 8 locations and selected with consideration to adaptation, diversity, and replication of alleles. Except for the set of common hybrids, testers included PHB47, PHZ51, LH82 and LH195. For the inbred experiments, 32 inbred lines were replicated twice over 17 locations in 25 experiments for a total of 1,600 plots.

## 2016 Experiment
In 2016, the project continued to expand to Arkansas, Michigan, and South Carolina. In total, 40 hybrid experiments across 34 unique locations in 19 states and one Canadian province were overseen by 29 principal investigators. Significant growth in both the number of experiments, and the addition of several “mega locations” in Iowa, Wisconsin, and Missouri, resulted in 19,360 plots planted in 2016. New in 2016, the hybrid experiment was organized into five broad sub-experiments, summarized below:

Ex-PVP Factorial ("Design 2") Experiment: Planned ex-PVP diallels integrated with genomic prediction based on ex-PVP-derived progeny. Three sets of materials selected for appropriate maturity were planted in 28 locations. Led by Jode Edwards (USDA ARS), Elizabeth Lee (Guelph), and Martin Bohn (University of Illinois Urbana-Champaign).
Stiff Stalk GxE: Founder lines, public early releases, and ex-PVP lines, which represent a span of selection pressure, crossed by inbred line 3IIH6. The Stiff Stalk Set was planted in 17 locations. Led by Natalia de Leon and Shawn Kaeppler (University of Wisonsin-Madison).
Near-Isogenic Lines Experiment: Teosintes introgressions; B73/Mo17 near-isogenic lines (NILs). The NILs experiment was planted in 17 locations. Led by Sherry Flint-Garcia (USDA-ARS) and Candice Hirsch (University of Minnesota).
Germplasm Enhancement of Maize (GEM): A highly diverse set of materials containing un-adapted, exotic germplasm to enhance genetic diversity and trait performance, and to minimize risks to production. The GEM lines were crossed with inbred line LH195 and were planted in 17 locations.
Common Hybrid Checks: A set of 50 common check hybrids and five local checks were planted in all locations. Two smaller sets of regional common hybrids (15 early and 5 late) were planted in locations based on maturity. Hybrid checks were selected for diversity and continuity with 2014-2015 experiments.
## 2017 Experiment
In 2017, the project will progress further with the addition of three new principal investigators in Arizona, Colorado, and Mississippi. As a result, over 20,000 plots comprising 41 experiments are being grown by 30 investigators in 37 unique locations.The experiment in 2017 will replicate the five broad sub-experiments planted in 2016, with the addition of a sixth sub-experiment headed by Randy Wisser (University of Delawaree).

## 2018/2019 Experimental Plans
For planting seasons 2018 and 2019, experiments will utilize genetic materials with a relatively narrow maturity window, across all locations; thus, resulting in reduced impact of flowering time on results, presenting more uniform data, and simplifying plot management for the collaborators.

### Experimental Design:

Randomized complete block design (RCBD)
### Populations:

PHW65 reference design population which includes double haploids derived from PHW65 X PHN11, PHW65 X Mo44, and PHW65 X MoG
A subset of ~30 hybrids per location will be repeated from the common set included in previous years.
### Testers:

LH195 (B37 X B73) Midwest to Southern locations
Early SS (TBD) for Northern locations
## Hybrids Used
[Download the full list](http://www.genomes2fields.org/docs/G2F-Lines-used-2014-2017.csv)  of hybrids used in 2014, 2015, 2016, and 2017

DNA sequence data from 858 inbred parental lines was derived using Genotyping by Sequencing (GBS) techniques. An additional 206 inbred lines will be sequenced in spring 2016 to complete the set of parental inbreds for all hybrids included in experiments in 2014-2016.

## Environmental Data Collected
Each experiment is furnished with a WatchDog 2700 Weather Station by Spectrum Technologies, Inc. The weather stations record air temperature, humidity, solar radiation, rainfall, wind speed and direction, soil temperature, and soil moisture every 30 minutes for the duration of the growing season. In addition, collaborators submit soil samples for basic nutrient and texture analysis to a central soil testing lab.

## Phenotypic Traits
The following table summarizes phenotypic traits measured by each GxE experiment cooperator at each location.

|Trait|	Abbrv.|	Unit|	Timing|	Description/Procedure|
|:----|:-----:|:----|:--------|:---------------------|
|Green Snap (optional)|	GSP|	Count and date of causal event [DD/MM/YY]|	Before flowering|	Number of plants broken between ground level and top ear node before flowering
|Anthesis	|DMF	|Date [DD/MM/YY]	|At flowering	|Days between planting and 50% of plants of a plot exhibit anther exertion on more than half of the main tassel spike
|Silking	|DFF	|Date [DD/MM/YY]	|At flowering	|Days between planting and 50% of plants of a plot show silk emergence
|Ear Height	|EHT	|Centimeter [cm] and date measured [DD/MM/YY]	|After flowering	|Placing measuring stick on ground next to the root crown, "ear height" is measured at the primary ear bearing node
|Plant Height	|PHT	|Centimeter [cm] and date measured [DD/MM/YY	|After flowering	|Measure the distance between the base of a plant and the ligule of the flag leaf
|Root Lodging	|RLD	|Count	|before harvest	|Number of plants that show root lodging per plot, i.e., those stems that lean substantially to one side (≥15% from vertical). Count includes "goosenecked" plants that have "straightened up" after becoming lodged earlier in the season
|Stalk Lodging	|SLD	|Count	|Before harvest	|Number of plants broken between ground level and top ear node at harvest
|Stand Count	|STC	|Count	|At harvest	|Number of plants per plot at harvest.
|Plot Weight	|PWT	|lbs	|At harvest	|Shelled grain weight per plot
|Grain Moisture	|GMT	|Percent [%]	|At harvest	|Water content in grain at harvest
|Test Weight	|TWT	|lbs/bu	|At harvest	|Shelled grain weight per bushel
